clcache.py

```diff
@@ -1507,10 +1507,7 @@ def main():
     if "CLCACHE_DISABLE" in os.environ:
         return invokeRealCompiler(compiler, sys.argv[1:])[0]
     try:
-        exitCode, compilerStdout, compilerStderr = processCompileRequest(cache, compiler, sys.argv)
-        printBinary(sys.stdout, compilerStdout.encode(CL_DEFAULT_CODEC))
-        printBinary(sys.stderr, compilerStderr.encode(CL_DEFAULT_CODEC))
-        return exitCode
+        return processCompileRequest(cache, compiler, sys.argv)
```

Comment:
Why did you need to do this? `processCompileRequest` now duplicates the code for printing the output in two places.